data/npclua/klom_stonecutter.lua

```diff
@@ -53,16 +53,21 @@ end
 npcConfig.voices = {
 	interval = 15000,
 	chance = 50,
-	{text = 'abc'} }
-local quantidade = {}
+	{text = 'abc'}
```

Comment:
```suggestion
	{text = 'Ah.'},
	{text = 'We need more volunteers!'},
	{text = 'And they call this "deep"...'},
	{text = 'Preparation is paramount.'},
```

Source: https://tibia.fandom.com/wiki/Klom_Stonecutter